nesdb/management/commands/nes_sync.py

```
@@ -0,0 +1,55 @@
+import subprocess
+from pathlib import Path
+
+from django.conf import settings
+from django.core.management.base import BaseCommand, CommandError
+
+from nesdb.importer import import_incremental
+
+
+def _is_inside_git_worktree(path: Path) -> bool:
+    try:
+        subprocess.run(
```

Ruff (0.15.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)